### PR TITLE
Serialize and deserialize AuthnEvent from session

### LINKED ISSUE #481: Session authn_event is not AuthnEvent instance
Usage :
https://github.com/OpenIDC/pyoidc/blob/8dca0d25ea618b10b6be1fe30b1a3a89766387c9/src/oic/oic/provider.py#L753-L755

But in session stored dict:
https://github.com/OpenIDC/pyoidc/blob/8dca0d25ea618b10b6be1fe30b1a3a89766387c9/src/oic/utils/sdb.py#L546-L555

So we will get crash.

### PR DESCRIPTION
**Pull request by tpazderka**
Close #481

- [x] Any changes relevant to users are recorded in the `CHANGELOG.md`.
- [ ] The documentation has been updated, if necessary.
---

### CODE REVIEW

File: src/oic/utils/sdb.py [hunk 1]
```diff
@@ -611,7 +619,12 @@ def create_authz_session(self, aevent, areq, id_token=None, oidreq=None,
         return sid
 
     def get_authentication_event(self, sid):
-        return self._db[sid]["authn_event"]
+        """Return AuthnEvetn based on sid."""
```
Comment (schlenk) [inline src/oic/utils/sdb.py]: AuthnEvent instead of AuthnEvetn

File: src/oic/utils/sdb.py [hunk 2]
```diff
@@ -657,9 +670,12 @@ def upgrade_to_token(self, token=None, issue_refresh=False, id_token="",
             dic["oidreq"] = oidreq
 
         if issue_refresh:
-            authn_event = dic.get('authn_event')
+            if 'authn_event' in dic:
+                authn_event = AuthnEvent.from_json(dic.get('authn_event'))
```
Comment (schlenk) [inline src/oic/utils/sdb.py]: why use dic.get() here when checking for existance first anyway?

**Comment by tpazderka** [comment]
Fixed

**Comment by tpazderka** [comment]
Merged

**Pull request closed without merging**